CHANGELOG.md

```diff
@@ -12,9 +12,19 @@ Bugfixes:
 
 Other improvements:
 
+## [v6.0.0](https://github.com/purescript-node/purescript-node-fs/releases/tag/v6.0.0) - 2021-MONTH-DAY
+
+Breaking changes:
+  - Updated dependencies for PureScript 0.14 (#46)
+  - Dropped deprecated `globals` dependency (#47)
+
+Other improvements:
+  - Migrated CI to GitHub Actions, updated installation instructions to use Spago, and migrated from `jshint` to `eslint` (#45)
+  - Added a changelog and pull request template (#49)
+
```

Comment:
Harry fixed the tests in [this commit](https://github.com/purescript-node/purescript-node-fs/commit/3da61c95b0542fc65c67f086f56f787aebbb03f4), but it looks like that was pushed straight to `master` rather than via a PR. Should we make note of that in the changelog and refer to the commit or leave it as is? Otherwise, looks good to me.

Reply:
I think it's fine to leave it out, but I'd appreciate a confirmation from @hdgarrood.

Reply:
I am good with leaving this out 👍